float32 time
# joint torques
float32 tau1
float32 tau2